person.rb

```diff
@@ -25,6 +26,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
```

Comment:
- [ ] To create a new rental, achieving the `has-many` association, the person itself should be sent as a parameter. Let's check out this below:

```ruby
   def add_rental(book, date)
  
         Rental.new(date, book, self)
   end
```

- This [Link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) may be beneficial to you.